R/processes.R

```diff
@@ -169,8 +183,8 @@ create_processes <- function(
     ),
     create_variable_mean_renderer_process(
       renderer,
-      c('ica', 'icm', 'ib', 'id', 'iva', 'ivm'),
-      variables[c('ica', 'icm', 'ib', 'id', 'iva', 'ivm')]
+      c('ica', 'icm', 'ib', 'id', 'iva', 'ivm', if(parameters$parasite == "vivax"){'idm'}),
```

Comment:
Would this be more clear (not checked):
```
variable_names <- c('ica', 'icm', 'ib', 'id', 'iva', 'ivm')
if(parameters$parasite == "vivax"){
  variable_names <- c(variable_names, "idm")
}
variable_names, 
variables[variable_names]
```

Reply:
I'll change this, but the section will need to come out of the processes list itself. Maybe that's fine.